roles/edpm_bootstrap/tasks/bootstrap.yml

```diff
@@ -160,3 +160,47 @@
       ansible.builtin.service:
         name: NetworkManager
         state: reloaded
+
+- name: Configure swap file
+  become: true
+  when:
+    - not edpm_bootstrap_swap_partition|bool
+    - edpm_bootstrap_swap_size_megabytes|int > 0
+  shell: |
+    #!/bin/bash
+    set -eu
+    if [ ! -f {{ edpm_bootstrap_swap_path }} ]; then
+      dd if=/dev/zero of={{ edpm_bootstrap_swap_path }} count={{ edpm_bootstrap_swap_size_megabytes }} bs=1M
+      chmod 0600 {{ edpm_bootstrap_swap_path }}
+      mkswap {{ edpm_bootstrap_swap_path }}
+      swapon {{ edpm_bootstrap_swap_path }}
+      if ! grep -qE "{{ edpm_bootstrap_swap_path }}\b" /etc/fstab; then
```

Comment:
Can't we use the native module to manipulate fstab entries ?

Reply:
Those are valid points, just please note that this change ports the unmodified implementation that we used to have in tripleo, and still do have in the layered product (where this feature is a release blocker for the upcoming release relying now on EDPM). This implementation has been suppored. We need to unblock the upcoming release by porting it, then we can think on changing it